### PR TITLE
GitHub folder

### PR DESCRIPTION
**Pull request by asclines**
This PR brings in two files: `CONTRIBUTING.MD` and `PULL_REQUEST_TEMPLATE.md`. Both in the `.github` folder.
The purpose of this PR is to bring instruction and uniformity to the process of contributing to this project, both for maintainers and for outside contributors. 
The important part of this PR that I urge reviewers to look at is in `CONTRIBUTING.md` the section titled "Getting Your Code Picked". It is here that I have outlined how **I believe** would be the best way to outside contributors (aka the people in the Applied Algorithms group) can submit their solutions to this repo. 

If y'all wanna look this over and add your two cents that would be much appreciated.

### CODE REVIEW

**Review by orionkoepke — APPROVED** [review]
Looks good to me.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributing
+So you want to contribute to this repository?
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners of this repository before making a change.
```
Comment (NilsG-S) [inline .github/CONTRIBUTING.md]: You use a very slight variation of "contribute to this repository" from the previous sentence.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Coding Standards
+Try to make it clean and easy to read with comments as necessary. A `README.md` is required for every problem solution. Please follow the standards put forth in the Templates folder.
```
Comment (NilsG-S) [inline .github/CONTRIBUTING.md]: "Try to make it clean" might be more clear if you said "try to make your code clean".

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Not all solution code will be clean. But make it as clean as possible within reason. The exact level of clean is up to the reviewers.
```
Comment (NilsG-S) [inline .github/CONTRIBUTING.md]: You use the word "clean" in three consecutive sentences.

**Review by NilsG-S — APPROVED** [review]
I don't see any serious problems, so I'm just going to nit pick. Feel free to ignore my suggestions.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (asclines) [reply .github/CONTRIBUTING.md]: Lol so line 2 was me, line 4 was @ASAAR from previous work. I'll amend that.

**Pull request merged**